pandas/core/generic.py

```diff
@@ -1940,6 +1940,77 @@ def tail(self, n=5):
             return self
         return self.iloc[-n:]
 
+  
+    def sample(self, n=None, frac=None, replace=False, weights=None, seed=None):
+        """
+        Returns a random sample of rows from object. 
+        
+        Parameters
+        ----------
+        n : int, optional
+            Number of rows to return. Cannot be used with frac.
```

Comment:
Can you do backticks around frac: like `` `frac` `` (the it is more clear it is a keyword argument, the same for `n` two lines below)